src/nmodl/nocpout.cpp

```diff
@@ -3046,7 +3061,9 @@ void net_receive(Item* qarg, Item* qp1, Item* qp2, Item* qstmt, Item* qend) {
         "  auto* const _ml = &_ml_real;\n"
         "  size_t const _iml{};\n");
     q = insertstr(qstmt, "  _ppvar = _nrn_mechanism_access_dparam(_pnt->_prop);\n");
-    vectorize_substitute(insertstr(q, ""), "  _thread = nullptr; _nt = (NrnThread*)_pnt->_vnt;");
+    vectorize_substitute(
+        insertstr(q, ""),
+        "  _thread = nullptr; double* _globals = nullptr; _nt = (NrnThread*)_pnt->_vnt;");
```

Comment:
Correct, because thread-variable aren't supported in NET_RECEIVE blocks or anything called from NET_RECEIVE blocks, because prior to this PR it would result in a `nullptr` dereference for `_thread[_gth]`.